libindi/drivers/telescope/celestrongps.cpp

```diff
@@ -351,11 +386,32 @@ bool CelestronGPS::updateProperties()
         // Sometimes users start their mount when it is NOT yet aligned and then try to proceed to use it
         // So we check issue and issue error if not aligned.
         checkAlignment();
+
+        //  handle the focuser
+        if (fwInfo.hasFocuser)
+        {
+            LOG_INFO("update focuser properties");
```

Comment:
This is probably not required.

Reply:
That code is intended to only show the focuser tab if a focuser exists, it seems to work but is there a better way?

Reply:
I meant just the part about LOG_INFO(...), probably the user does not need to see this message. You can make it LOG_DEBUG(...) for example or just comment it.

Reply:
I tend to be a bit too generous with log messages, I'll make it debug for the next build but won't commit specially.